src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSServiceHelper.cs

```diff
@@ -20,90 +20,47 @@ public AWSServiceHelper(AWSSemanticConventions semanticConventions)
                 .AddAttributeAWSBedrockDataSourceId("DataSourceId")
                 .AddAttributeAWSBedrockGuardrailId("GuardrailId")
                 .AddAttributeAWSBedrockKnowledgeBaseId("KnowledgeBaseId")
+                .AddAttributeAWSSQSQueueName("QueueName")
+                .AddAttributeAwsS3Bucket("BucketName")
+                .AddAttributeAwsKinesisStreamName("StreamName")
+                .AddAttributeAwsSnsTopicArn("TopicArn")
+                .AddAttributeAwsSecretsmanagerSecretArn("ARN")
+                .AddAttributeAwsSecretsmanagerSecretArn("SecretId")
+                .AddAttributeAwsStepFunctionsActivityArn("ActivityArn")
+                .AddAttributeAwsStepFunctionsStateMachineArn("StateMachineArn")
+                .AddAttributeAwsLambdaResourceMappingId("UUID")
+                .AddAttributeAWSLambdaFunctionName("FunctionName")
+                .AddAttributeAWSKinesisStreamArn("StreamArn")
+                .AddAttributeAWSDynamoTableArn("TableArn")
+                .AddAttributeAWSBedrockGuardrailArn("GuardrailArn")
                 .Build();
     }
 
     internal static IReadOnlyDictionary<string, List<string>> ServiceRequestParameterMap { get; } = new Dictionary<string, List<string>>()
     {
-        { AWSServiceType.DynamoDbService, ["TableName"] },
-        { AWSServiceType.SQSService, ["QueueUrl"] },
+        { AWSServiceType.DynamoDbService, ["TableName", "TableArn"] },
+        { AWSServiceType.SQSService, ["QueueUrl", "QueueName"] },
         { AWSServiceType.BedrockAgentService, ["AgentId", "KnowledgeBaseId", "DataSourceId"] },
         { AWSServiceType.BedrockAgentRuntimeService, ["AgentId", "KnowledgeBaseId"] },
         { AWSServiceType.BedrockRuntimeService, ["ModelId"] },
+        { AWSServiceType.S3Service, ["BucketName"] },
+        { AWSServiceType.KinesisService, ["StreamName", "StreamArn"] },
```

Comment:
The parameter should be `StreamARN` instead of `StreamArn`.
https://github.com/aws/aws-sdk-net/blob/main/sdk/src/Services/KinesisVideo/Generated/Model/DescribeStreamRequest.cs#L49